src/govuk/common.unit.test.mjs

```diff
@@ -0,0 +1,103 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env jest */
+
+import { mergeConfigs, extractConfigByNamespace } from './common.mjs'
+
+// TODO: Write unit tests for `nodeListForEach` and `generateUniqueID`
+
+describe('Common JS utilities', () => {
+  describe('mergeConfigs', () => {
+    const config1 = {
+      a: 'antelope',
+      c: { a: 'camel' }
+    }
+    const config2 = {
+      a: 'aardvark',
+      b: 'bee',
+      c: { a: 'cat', o: 'cobra' }
+    }
+    const config3 = {
+      b: 'bat',
+      c: { o: 'cow' },
+      d: 'dog'
+    }
+
+    test('flattens a single object', () => {
```

Comment:
Minor, but is there a reason we're using `test` here when we use `it` everywhere else?

Reply:
Ollie also [left the same comment on my PR](https://github.com/alphagov/govuk-frontend/pull/2819/files#r960376063), so I thought I'd copy in my response here because I'm pretty sure this originated from my unit test spike!

> Good point 🤔 There's a bit of info on [the difference here](https://stackoverflow.com/a/56072272/1669682), but I think `it` makes the most sense in terms of 1) what we're doing here, and 2) to be consistent with our other tests. 

> I think the only reason it's crept in is because it's what [Jest uses in it's Getting Started guide](https://jestjs.io/docs/getting-started). Will amend now!

Reply:
Yeah, I was just copying the tests from the spike. Happy to make these into `it`s instead.